css21/visudet/height-computed-001.xht

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+
+<html xmlns="http://www.w3.org/1999/xhtml">
+
+ <head>
+  <title>CSS min-height: min-height does not influence the computed height</title>
+
+  <link rel="help" href="http://www.w3.org/TR/CSS22/visudet.html#min-max-heights" title="10.7 Minimum and maximum heights: 'min-height' and 'max-height'" />
+  <meta name="assert" content="If the resulting height is smaller than 'min-height', the rules above are applied again, but this time using the value of 'min-height' as the computed value for 'height'. Note: These steps do not affect the real computed value of 'height'." />
+  <link rel="match" href="height-computed-001-ref.xht" />
+  <link rel="author" title="Bert Bos" href="mailto:bert@w3.org" />
+
+  <meta name="flags" content="" />
+
+  <style type="text/css"><![CDATA[
+  body {background: white; color: black; font-size: 14px}
+  span {display: inline-block}
+  #container {background: green; height: 70px; min-height: 140px}
+  #child1 {background: #66F; width: 70px; height: inherit} /* Inherits 70px */
+  #child2 {background: #AAF; width: 70px; height: 70px}
+  ]]></style>
+ </head>
+
+ <body>
+   <p>Test passes if (1) the two blue rectangles have the same height
```

Comment:
In general, we've used teal/orange boxes, rather than two of similar colour. (Likely this isn't that essential, but…)

I'd also recommend not having green there as a signifier, rather having a 140x70 red box *under* the other two, and then have the condition that there should be no red.